compass/ocean/tests/spherical_harmonic_transform/QU_convergence/mesh.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+
+from mpas_tools.ocean import build_spherical_mesh
+from mpas_tools.logging import check_call
+
+from compass.step import Step
+from compass.model import make_graph_file
+
+import subprocess
+import os
+
+
+class Mesh(Step):
+    """
+    A step for creating uniform global meshes
+
+    Attributes
+    ----------
+    resolution : int
+        The resolution of the (uniform) mesh in km
+    """
+    def __init__(self, test_case, resolution, serial_nLat):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.spherical_harmonic_transform.QU_convergence
+            The test case this step belongs to
+
+        resolution : int
+            The resolution of the (uniform) mesh in km
+        """
+        super().__init__(test_case=test_case,
+                         name='QU{}_mesh'.format(resolution),
+                         subdir='QU{}/mesh'.format(resolution))
```

Comment:
Here or elsewhere, you should give a minimum and a target number of cores so `ESMF_RegridWeightGen` can run in parallel.  Also, please use f-strings instead of the `.format()` method.  I know there's plenty of code that doesn't do that right in `compass` but I'm trying to fix it as I see it.
```suggestion
        super().__init__(test_case=test_case,
                         name=f'QU{resolution}_mesh',
                         subdir=f'QU{resolution}/mesh',
                         cores=18, min_cores=1, threads=1)
```

Reply:
BTW, 18 cores is just a (fairly arbitrary) suggestion.